dags/python_scripts/cr3_extract_diagram_ocr_narrative.py

```diff
@@ -153,8 +175,9 @@ def update_crash_metadata(crash_id: int, metadata: dict) -> bool:
   }
 }
 """
-batch_variables = { "limit": int(args.batch_size) }
+batch_variables = {"limit": int(args.batch_size)}
 
+# query for specific cr3, selected by crash_id
 get_single_cr3 = """
 query find_cr3s($crash_id: Int, $limit: Int) {
```

Comment:
I was considering editing this query to remove the $limit var, since this will only ever return 1 record, but I decided against refactoring code that wasn't related to the OCR.